### PR TITLE
feat: live testnet improvements

### PR DESCRIPTION
**Pull request by deanamiel**
Testing improvements for live testnets as per [AXE-2831](https://axelarnetwork.atlassian.net/jira/software/c/projects/AXE/boards/21?selectedIssue=AXE-2831)

[AXE-2831]: https://axelarnetwork.atlassian.net/browse/AXE-2831?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: scripts/deploy.js [hunk 1]
```diff
@@ -118,7 +118,19 @@ async function deployAll(
         interchainTokenServiceAddress,
         factoryDeploymentKey,
     );
-    return [service, gateway, gasService, factory];
+
+    return [
```
Comment (milapsheth) [inline scripts/deploy.js:122]: use an object instead so order doesn't matter?

**Comment by milapsheth** [comment]
can you also move any contract ABI imports of the form `../artifacts/..` to use the get factory helper to avoid hardcoding the path?

**Comment by deanamiel** [comment]
> can you also move any contract ABI imports of the form `../artifacts/..` to use the get factory helper to avoid hardcoding the path?

Yeah good idea

File: test/InterchainTokenService.js [hunk 2]
```diff
@@ -651,47 +624,33 @@ describe('Interchain Token Service', () => {
...
-                .to.emit(service, 'TokenManagerDeployed')
-                .withArgs(tokenId, expectedTokenManagerAddress, MINT_BURN, params);
-            const tokenManagerAddress = await service.validTokenManagerAddress(tokenId);
-            expect(tokenManagerAddress).to.not.equal(AddressZero);
-            const tokenManager = await getContractAt('TokenManager', tokenManagerAddress, wallet);
-
-            expect(await tokenManager.hasRole(wallet.address, OPERATOR_ROLE)).to.be.true;
+        it('Should revert when registering an interchain token when service is paused', async () => {
+            txPaused = await service.setPauseStatus(true);
+            await txPaused.wait();
```
[the remark this answers is not in the document]
Comment (deanamiel) [reply test/InterchainTokenService.js]: updated

**Review by milapsheth — APPROVED** [review]

**Pull request merged**